technical-vision.md

```diff
@@ -0,0 +1,9 @@
+# OpenSSF Technical Vision
+
+We envision a future where participants in the open source software ecosystem focus on delivering high quality products and services, with security handled proactively, automatically, and as a matter of course.
```

Comment:
could we change products to projects, open source is not always a product, but always projects (even when someone creates a product out of them). Perhaps drop services (if this is meant in the 'professional services' capacity), if its 'to serve' then we just need to tweak the text. 

For the rest, I think the statement on 9 needs to come first or we reword, as otherwise the prior sentences sound like current facts. 

>  Our vision at The Open Source Security Foundation (OpenSSF) is to provide tools, services, training, infrastructure, and resources to help:

> - Developers learn secure development practices, and are proactively guided by their tools to apply those practices and automatically informed when action is needed to prevent, remediate, or mitigate security issues.
> - Developers, auditors and regulators to create and easily distribute security policies that are enforced through tooling and automation, providing continuous assurance of the results.

One last nit, there might be a better word them stakeholders, I much prefer "The community"

Reply:
One comment here -- I don't think it's the intent, but the language might be taken to imply that "all" security work is done "automatically", or that "the only security work that is done will be driven from automated tools". I think the intent is (rightly) that "tools/technologies/platforms are used to detect, remediate, or obviate as much security risk as possible", or something to that extent.

Reply:
How about "We envision a future where participants in the open source software ecosystem focus on delivering high quality products and services, with security handled proactively and as a matter of course."  This reduces the focus on automation. I prefer not to narrow the scope to 'projects' since the broader ecosystem includes projects, packages, products, services, etc.  We could use a word like 'offerings' to encompass them all.  But for a general audience, I think the phrase 'products and services' is more natural wording.

Reply:
I still think we need to remove products, as a lot of the eco-system are not products. They are software projects started as a labour of love , which have evolved into wide use (including the mother of them all, Linux). Perhaps I am getting hung up on semantics. 

I think this should solve it..

> "We envision a future where participants in the open source software ecosystem focus on delivering high quality secure code and services, with security handled proactively and as a matter of course."

Reply:
I agree with Luke on this. In the context of open source, either 'project' and/or 'code' is more appropriate as not all OSS will be a product or a service. It can be a tool, a component or library. A project/code is all encompassing, 'product' is narrow IMO.

Reply:
How about this wording? 

> "We envision a future where participants in the open source ecosystem use and share high quality software, with security handled proactively and as a matter of course:"

Here we use the general word 'software'.